omero_figure/views.py

```diff
@@ -543,6 +544,50 @@ def unit_conversion(request, value, from_unit, to_unit, conn=None, **kwargs):
     return HttpResponse(json.dumps(rsp), content_type='application/json')
 
 
+@login_required()
+def roi_rectangles(request, image_id, conn=None, **kwargs):
```

Comment:
is not something that should be pushed to the API itself with something like ?type=...

Reply:
Yes that would be great, but that won't likely happen in the timeframe of this PR.

Support for `/api/rois/?class=Rectangle` could work (uses same terminology as the HQL). But we'd need a custom mapping for that URL to say "class applies to Shapes", whereas e.g. `/api/annotations/?class=TagAnnotation` the class would apply to the annotations themselves. Writing query-builder code that doesn't use custom mapping might use a notation like `/api/rois/?shapes.class=Rectangle`.

I can imagine that work taking a while, and I wouldn't want that to hold up this PR. But we can certainly revert to using the API once it's supported there.

Reply:
we have explored in the past in "clients" possible API call.
The key point is to make sure we go back to it when the API call is implemented. 
Could you create an issue for the API call addition and figure update

Reply:
Issue created at https://github.com/ome/omero-web/issues/258